### PR TITLE
add default values

### PR DESCRIPTION
**Pull request by tim-thacker-nullify**
## Description

Created a public function to return the default values if the user doesn't set them.
The ParseConfiguration function also uses this to set default values in parsed config files.

## Linked issues

- https://github.com/Nullify-Platform/The-Force/issues/576

## Checklist

- [x] I have added one of the `patch`, `minor`, `major` or `no-release` labels
- [x] I have performed a self-review of my own code
- [x] I have checked for redundant or commented out code
- [x] I have commented my code where I can't make it self-documenting
- [ ] I have made corresponding changes to the documentation
- [x] I have added any appropriate tests

### CODE REVIEW

File: pkg/models/severity.go [hunk 1]
```diff
@@ -5,4 +5,5 @@ const (
 	SeverityHigh     = "HIGH"
 	SeverityMedium   = "MEDIUM"
 	SeverityLow      = "LOW"
+	SeverityUnknown  = "UNKNOWN"
```
Comment (yushindkang) [inline pkg/models/severity.go:8]: @willNullify looks like `SeverityUnknown` is never used

Comment (willNullify) [reply pkg/models/severity.go:8]: need it for the-force

File: pkg/parser/parse.go [hunk 2]
```diff
@@ -1,15 +1,25 @@
...
 		return nil, err
 	}
-	return &config, nil
+	sanitizeConfig(config)
+	return config, nil
+}
+
+func sanitizeConfig(config *models.Configuration) {
+	config.MinimumCommentSeverity = strings.ToUpper(config.MinimumCommentSeverity)
+	if config.MinimumCommentSeverity == "" {
```
Comment (yushindkang) [inline pkg/parser/parse.go:22]: @willNullify is this redundant? We're setting the `MinimumCommentSeverity` to medium in `NewDefaultConfig()`

File: pkg/models/severity.go (hunk 1 above)
Comment (yushindkang) [reply pkg/models/severity.go:8]: @tim-thacker-nullify

File: pkg/parser/parse.go (hunk 2 above)
Comment (yushindkang) [reply pkg/parser/parse.go:22]: @tim-thacker-nullify

File: pkg/parser/parse_test.go [hunk 3]
```diff
@@ -3,19 +3,55 @@ package parser
...
 `
 
-func TestParseConfiguration(t *testing.T) {
-	config, err := ParseConfiguration([]byte(configStr))
-	require.NoError(t, err)
-	assert.Equal(t, "medium", config.MinimumCommentSeverity)
-	assert.Equal(t, 1, len(config.IgnoreDirs))
-	assert.Equal(t, "data", config.IgnoreDirs[0])
+func TestValidateMinimumCommentSeverity(t *testing.T) {
+	for _, scenario := range []struct {
```
Comment (yushindkang) [inline pkg/parser/parse_test.go:17]: @tim-thacker-nullify can we add a test case where use provides 'LOW' minimum severity

File: pkg/models/severity.go (hunk 1 above)
Comment (tim-thacker-nullify) [reply pkg/models/severity.go:8]: yeah its used by the services

**Review by yushindkang — APPROVED** [review]
